affinity/cpp-20/d0796r2.md

```diff
@@ -515,6 +566,20 @@ The free function `this_system::resources` is provided for retrieving the `execu
 
 > [*Note:* Returning a `std::vector` allows users to potentially manipulate the container of `execution_resource`s after it is returned, we may want to replace this with an alternative type which is more restrictive at a later date such as a range. *--end note*]
 
+The free functions `this_thread::bind` and `this_thread::unbind` are provided for binding / unbinding the current *thread of execution* to / from a particular `execution_reosurce`.
+
+    bool bind(executon_resource) noexcept;
+
+*Returns:* `true` if the requested binding was successful, otherwise `false`.
+
+*Effects:* If successful, binds the current *thread of execution* to the specified `execution_resource`.
+
+    bool unbind(executon_resource) noexcept;
+
+*Returns:* `true` if the requested unbinding was successful, otherwise `false`.
```

Comment:
Is true and false enough information on the error? is there any particular reason for not  to throw an exception?